### PR TITLE
OCRVS-2381: List view simplified on settings page

### PR DESCRIPTION
**Pull request by emon-swe-sust**
settings_simplified

### CODE REVIEW

File: packages/client/src/i18n/messages/user.ts [hunk 1]
```diff
@@ -458,6 +462,33 @@ const messagesToDefine: IUserMessages = {
...
+    id: 'imageUploadOption.upload.imageFormat',
+    defaultMessage:
+      'Image format not supported. Please attach a png or jpg (max 5mb)',
+    description:
+      'Show error message if the selected image type is not supported'
+  },
+  overSized: {
+    id: 'imageUploadOption.upload.overSized',
+    defaultMessage: 'File is too large. Please attach file less than 5mb',
+    description: 'Error message for Attachment size greater than 5mb.'
```
Comment (Zangetsu101) [inline packages/client/src/i18n/messages/user.ts:491]: These might have been duplicated

File: packages/client/src/views/Settings/SettingsPage.tsx [hunk 2]
```diff
@@ -312,120 +289,103 @@ class SettingsView extends React.Component<IProps, IState> {
...
+                )
+              })}
+            </ListViewSimplified>
+            <HiddenInput
+              ref={this.fileUploaderRef}
+              id="image_file_uploader_field"
+              type="file"
+              accept={ALLOWED_IMAGE_TYPE.join(',')}
+              onChange={this.handleSelectFile}
+            />
```
Comment (Zangetsu101) [inline packages/client/src/views/Settings/SettingsPage.tsx:387]: Maybe we can reuse `ImageLoader`

Comment (emon-swe-sust) [reply packages/client/src/views/Settings/SettingsPage.tsx]: Yeah, We can . I did that. Thank you for your suggestion.

File: packages/client/src/i18n/messages/user.ts (hunk 1 above)
Comment (emon-swe-sust) [reply packages/client/src/i18n/messages/user.ts]: Yeah, they are already in imageUploader messages.

File: packages/client/src/views/Settings/SettingsPage.tsx [hunk 3]
```diff
@@ -312,120 +258,113 @@ class SettingsView extends React.Component<IProps, IState> {
...
+                    onError={(imageLoadingError) =>
+                      this.setState({ imageLoadingError })
+                    }
+                  >
+                    <LinkButton>
+                      {intl.formatMessage(buttonMessages.change)}
+                    </LinkButton>
+                  </ImageLoader>
+                ]}
+              ></ListViewItemSimplified>
```
Comment (Zangetsu101) [inline packages/client/src/views/Settings/SettingsPage.tsx:365]:
```suggestion
              />
```

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]

**Review by Zangetsu101 — APPROVED** [review]

**Pull request merged**